### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by rhandberg**
Update all dependencies to newest supported version. Also remove old, no longer needed, dependencies

### CODE REVIEW

**Review by jeraud — COMMENTED** [review]
Code seems to run fine: no errors during training.
We will have to fix the following in the future though:
`/python3.7/site-packages/xgboost/sklearn.py:892: UserWarning: The use of label encoder in XGBClassifier is deprecated and will be removed in a future release. To remove this warning, do the following: 1) Pass option use_label_encoder=False when constructing XGBClassifier object; and 2) Encode your labels (y) as integers starting with 0, i.e. 0, 1, 2, ..., [num_class - 1].
  warnings.warn(label_encoder_deprecation_msg, UserWarning)`

**Comment by rhandberg** [comment]
Just pushed changes to XBG which eliminates the warning, and now uses integer labels.

**Comment by rhandberg** [comment]
@jeraud do you want to do another final test of the changes to XGB before merging, or am I allowed to merge this?

**Comment by jeraud** [comment]
> @jeraud do you want to do another final test of the changes to XGB before merging, or am I allowed to merge this?

I did a final test with xgb and everything seems to be running fine, so you can merge it.
